ramls/schemas/profiles/mappingProfile.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Mapping Profile schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "Unique identifier",
+      "$ref": "../../raml-util/schemas/uuid.schema"
+    },
+    "name": {
+      "description": "Mapping Profile name",
+      "type": "string"
+    },
+    "description": {
+      "description": "Mapping Profile description",
+      "type": "string"
+    },
+    "recordTypes": {
+      "description": "Mapping Profile records type",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "$ref": "recordTypes.json"
+      }
+    },
+    "mappingProfileRules": {
+      "description": "List of mapping profile rules",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "mappingProfileRule.json"
+      }
+    },
+    "userInfo": {
+      "description": "First name, last name and username of the user, who updated the Mapping Profile",
+      "type": "object",
+      "$ref": "userInfo.json"
+    },
+    "outputFormat": {
+      "description": "Mapping Profile output format",
+      "type": "array",
```

Comment:
Output format need not be an array, it will be just one type per mapping profile